base_exception/README.rst

```diff
@@ -29,6 +29,11 @@ Bugs are tracked on `GitHub Issues
 check there if your issue has already been reported. If you spotted it first,
 help us smash it by providing detailed and welcomed feedback.
 
+Roadmap
+-------
+Terms used in old api like `pool`, `cr`, `uid` must be removed porting this module in version 12.
```

Comment:
Is this comment still needed?

Reply:
This is about `'cr': self.env.cr,` and questioning the need of it in the context.

For backward compatibility we probably want to keep it if very complexe exceptions were created, otherwise it can be deprecated in v 12 as you can reach `cr` and `uid` using `env` you have in the context.